index.js

```diff
@@ -13,6 +13,8 @@ export default function toCSS(obj) {
       value = `${value}px`;
     }
 
-    return `${key}:${value} !important;`;
+    const important = 'ontouchstart' in document.documentElement ? '' : '!important';
```

Comment:
As far as I know this will tell _if_ a browser supports touch events, not if it's a touch device. So Chrome on Windows will return true for this because Windows technically can do touch and it tells Chrome as much. So Windows browsers might false positive on this.